src/usb/class/hid/hid.c

```diff
@@ -43,11 +43,11 @@ static int32_t _enable(struct usbd_descriptors* desc, struct usbdf_driver* drv)
     if (HID_CLASS == ifc_desc.bInterfaceClass) {
         if (func_data->func_iface == ifc_desc.bInterfaceNumber) { // Initialized
             return ERR_ALREADY_INITIALIZED;
-        } else if (func_data->func_iface != 0xFF) { // Occupied
```

Comment:
just wondering, this code is not by us, right? so maybe we shouldn't spend time on stylistic changes to this.

mainly asking because it's slightly hard to review.

speaking of: should we expect the compiled binary to stay exactly the same after this PR? that would make me more comfortable that everything is still working.

Reply:
This is written by us, otherwise I would've put it in the `external` folder.

Very unlikely. getting the order of the binaries the same might not even be possible. But the binary size should stay the same. So I'm pretty confident it is the same result if that happens. I don't really see what can go wrong anyway, if the thing compiles and runs I can't see how it would be broken.

Reply:
> This is written by us, otherwise I would've put it in the external folder.

This is also partly why I badly want this reorganization. It is hard as a newcomer to understand what is our code and not when it is all mixed up.